Ginger/GingerCoreCommon/UserCofig/UserProfile.cs

```diff
@@ -567,7 +567,7 @@ public string ProfileImage
 
 
         string mActionOutputValueUserPreferences;
-        [IsSerializedForLocalRepository(DefaultValue: "Description:False,Path:True,ActualValue:True,ExpectedValue:True,StoreTo:True")]
+        [IsSerializedForLocalRepository(DefaultValue: ",Path,ActualValue,ExpectedValue,StoreTo")]
```

Comment:
',' at start is required?

Reply:
yes because only those column are visible we are storing it's name on xml, otherwise not storing .
so by default we are not showing the description column that's why it's name is empty